### PR TITLE
Have the login node commit app ports as fast as possible.

### PR DESCRIPTION
**Pull request by menivaitsi**
https://ocd.appscale.com:8080/job/Daily%20Build/1365/

### CODE REVIEW

File: AppController/djinn.rb [hunk 1]
```diff
@@ -4950,6 +4950,29 @@ def setup_appengine_application(app, state)
       end
     end
 
+    if my_node.is_login?
+      loop {
+        begin
+          success = uac.add_instance(app, my_public, nginx_port, https_port)
```
Comment (cdonati) [inline AppController/djinn.rb]: Would it make sense to move this `uac.add_instance` call to `add_routing_for_appserver`?

Comment (menivaitsi) [inline AppController/djinn.rb]: The first occurrence could be moved in there.
The other one that's happening upon relocate doesn't go through the same path though.

**Comment by menivaitsi** [comment]
Demo done. Build pending.

**Comment by menivaitsi** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1375/

**Pull request merged**